src/public/modules/forms/services/mailto.client.factory.js

```diff
@@ -0,0 +1,42 @@
+const dedent = require('dedent-js')
+
+angular.module('forms').factory('MailTo', ['$window', MailTo])
+
+function MailTo($window) {
+  /**
+   * Generates a mailto URI which opens the client's default email application
+   * with an email to share their secret key. This requires the form title and
+   * secret key at minimum, and optionally the form ID.
+   * @param {string} title
+   * @param {string} secretKey
+   * @param {string} [formId]
+   */
+  const generateMailToUri = (title, secretKey, formId) => {
+    // Construct body piece by piece to avoid confusion with order of operations
+    let body = dedent`
+      Dear collaborator,
+
+      I am sharing my form's secret key with you for safekeeping and backup. This is an important key that is needed to access all form responses.
+
+      Form title: ${title}`
+    if (formId) {
```

Comment:
why would `formId` not be defined?

Reply:
we also use this function in the create form modal, but at that point the form has not been created yet so it doesn't have an ID, so the `formId` parameter is optional in this function

Reply:
got it